src/models/v3/message.ts

```diff
@@ -17,12 +17,18 @@ import type { ServerInterface } from '../server';
 import type { SchemaInterface } from '../schema';
 
 import type { v3 } from '../../spec-types';
+import { getDefaultSchemaFormat } from 'schema-parser';
 
 export class Message extends MessageTrait<v3.MessageObject> implements MessageInterface {
   hasPayload(): boolean {
     return !!this._json.payload;
   }
   
+  schemaFormat(): string | undefined {
+    const payloadIsSchemaUnion = this._json.payload.schemaFormat !== undefined;
+    return payloadIsSchemaUnion ? this._json.payload.schemaFormat : getDefaultSchemaFormat(this._meta.asyncapi.semver.version);
```

Comment:
```suggestion
    const payloadIsSchemaUnion = this._json.payload?.schemaFormat !== undefined;
    return payloadIsSchemaUnion ? this._json.payload.schemaFormat : getDefaultSchemaFormat(this._meta.asyncapi.semver.version);
```

I think `payload` can be undefined, right? We should be careful then.